docs/KarmaCommands.md

```diff
@@ -0,0 +1,60 @@
+# Karma Command Reference
+
+Command to giveth or taketh away a user's karma
+
+## Options
+
+### For normal users
+
+#### Add 1 karma to user
+
+```sh
+/rocket @user ++
```

Comment:
Would this just be the `elif` statement on `command/core.py:41`, to check if the user exists or not?

Reply:
Kinda. I was thinking of something like doing a regex match to see if it's a user id passed in then going to the karma command handlers to check if the user exists or not.